### DISCUSSION TITLE
How to know which GitHub account made a commit?

### DISCUSSION THREAD

**Discussion by valera-rozuvan** [community/community#21702] [New to GitHub]
<p>Suppose a company has a large private GitHub repository. There are 200 GitHub accounts allowed to commit there. If it is discovered that the Git history of that private repo contains commits with suspcious code, obviously they were made by someone from the 200 allowed GitHub accounts. What if the rogue commiter specified an e-mail not known to the company (for example “<a href="mailto:user@example.com">user@example.com</a>”) in the commit. How is it possible to discover original GitHub user of those rogue commits? From the GitHub UI, it is not possible to click on the user for that specific commit (from the list of all commits in a repo). Because the e-mail “<a href="mailto:user@example.com">user@example.com</a>” is not tied to any GitHub user account. The e-mail is just meta information in a commit.</p>

**Comment by valera-rozuvan** [comment]
<p>Up. This is still very relevant. Need help.</p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/valera-rozuvan">@valera-rozuvan</a>,</p>
<p>This seems like an account-specific question. I’d recommend reaching out to our private support team via our contact form <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>, so that we can better assist you.</p>
